bin/configure-tower-projects.py

```diff
@@ -556,6 +556,8 @@ def cleanup_compute_environments(self):
         for comp_env in response["computeEnvs"]:
             comp_env_id = comp_env["id"]
             comp_env_name = comp_env["name"]
+            if comp_env_name.endswith(CE_VERSION):
```

Comment:
What happens if a compute environment is in use?  Can it not be deleted?

Reply:
It cannot, and I'm relying on this fact since it would be complicated to replicate this behavior myself. 

For compute environments with running jobs, the `DELETE` request fails with:
```
 ERROR: Compute environment '1GZDXqNBbdeWFRdaWnGbKE' has active jobs
```